app/javascript/components/enrollment/steps/Address.js

```diff
@@ -20,6 +20,36 @@ class Address extends React.Component {
     let value = event.target.type === 'checkbox' ? event.target.checked : event.target.value;
     let current = this.state.current;
     let modified = this.state.modified;
+
+    const trimFields = [
+      'address_line_1',
+      'address_city',
+      'address_line_2',
+      'address_zip',
+      'address_county',
+      'monitored_address_line_1',
+      'monitored_address_city',
+      'monitored_address_line_2',
+      'monitored_address_zip',
+      'monitored_address_county',
+      'foreign_address_line_1',
+      'foreign_address_city',
+      'foreign_address_line_2',
+      'foreign_address_zip',
+      'foreign_address_line_3',
+      'foreign_address_state',
+      'foreign_monitored_address_line_1',
+      'foreign_monitored_address_city',
+      'foreign_monitored_address_line_2',
+      'foreign_monitored_address_zip',
+      'foreign_monitored_address_county',
+    ];
+    if (event?.target?.id && trimFields.includes(event.target.id)) {
```

Comment:
I dont really care one way or the other, but was there any specific reason for using an `if` within an `if` ? Not a huge deal, and it might even make it more legible, just wondering if it was an intentional decision.

Reply:
Thanks for the suggestion! Will push an commit with updated shortly.